
To prevent pkgdepend from screaming at us, remove the #! from appdirs.py.
This allows pkgdepend to correctly detect the utf-8 encoding required for
the copyright block.

Not submitting upstream because this is specific to IPS requirements.

--- setuptools-62.1.0/pkg_resources/_vendor/appdirs.py
+++ setuptools-62.1.0/pkg_resources/_vendor/appdirs.py
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # Copyright (c) 2005-2010 ActiveState Software Inc.
 # Copyright (c) 2013 Eddy Petri»ôor
